js/build.js

```diff
@@ -0,0 +1,344 @@
+Fliplet.Widget.instance({
+  name: 'file-list',
+  displayName: 'File list',
+  icon: 'fa-exchange',
+  render: {
+    dependencies: [],
+    template: [
+      '<div data-view="content"></div>',
+    ].join(''),
+    ready: async function () {
+      var thisy = this;
+      const populateFileList = (entryID = Fliplet.Navigate.query.dataSourceEntryId, dsID = thisy.fields.dataSource) => {
```

Comment:
Why not just do this instead of assigning them to new variables?

```suggestion
      const populateFileList = (dataSourceEntryId = Fliplet.Navigate.query.dataSourceEntryId, dataSourceId = thisy.fields.dataSource) => {
```

Reply:
Done